spec/features/todo_items/todo_items_spec.rb

```diff
@@ -15,6 +15,7 @@
     } }
 
   before(:each) do
+    user.update_attribute('activation_token', nil)
```

Comment:
Ha - i tu jest błąd ;) Lepszym rozwiązaniem było by zrobić 2 fabryki:
1) `unconfirmed_user` która nie ustawia tokena, więc ma go domyślnie wygenerowanego - po czym wykorzystać wszędzie gdzie testujesz rejestrację i potwierdzanie
2) `user` dziedziczące po `unconfirmed_user` która ma wspomniane `after_create` i wykorzystywane wszędzie indziej
Oczywiście można to zamienić - na `user` i `confirmed_user` - dzięki temu nie zapomnisz o dodawaniu `unconfirmed` w kluczowych miejscach - szczerze powiedziawszy nie wiem które zn ich jest lepsze :)